src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_help.py

```diff
@@ -284,6 +284,14 @@
     type: command
     short-summary: Update a server.
     """
+helps['sql server dns-alias'] = """
+    type: group
+    short-summary: Manage a server's DNS aliases.
+    """
+helps['sql server dns-alias set'] = """
+    type: command
+    short-summary: Sets a server to which DNS alias should point
```

Comment:
sounds a bit weird, how about "Changes the server that a DNS alias is pointing to."

Reply:
or "Sets the server that a DNS alias is pointing to."